gui/templates/channel.html

```diff
@@ -245,10 +245,10 @@ <h2>Last 5 <a href="/rebalancing" target="_blank">Rebalance Requests</a></h2>
     {% for rebalance in rebalances %}
     <tr>
       <td title="{{ rebalance.requested }}">{{ rebalance.requested|naturaltime }}</td>
-      <td {% if rebalance.status == 0 %}>N/A{% else %}title="{{ rebalance.start }}">{{ rebalance.start|naturaltime }}{% endif %}</td>
-      <td {% if rebalance.status > 1 %}title="{{ rebalance.stop }}">{{ rebalance.stop|naturaltime }}{% else %}>N/A{% endif %}</td>
+      <td {% if rebalance.status == 0 %}>{% else %}title="{{ rebalance.start }}">{{ rebalance.start|naturaltime }}{% endif %}</td>
+      <td {% if rebalance.status > 1 %}title="{{ rebalance.stop }}">{{ rebalance.stop|naturaltime }}{% endif %}</td>
```

Comment:
Here the end block in the html `>` is missing due to the `else` being removed.

This looks to occur in a few other places as well.

Reply:
checking

Reply:
https://github.com/cryptosharks131/lndg/pull/108/commits/92ddd62238f02eca2f53f88e4f023bc3ba0adf43